Volume/Views/MainView and Tabs/FlyersView.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  FlyersView.swift
+//  Volume
+//
+//  Created by Vin Bui on 4/2/23.
+//  Copyright © 2023 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct FlyersView: View {
+    var body: some View {
+        Text("Flyers View")
+    }
+}
+
+struct FlyersView_Previews: PreviewProvider {
```

Comment:
Not sure how casing works in SwiftUI but did you mean to make this camel case? Or too much Ocaml 😔

Reply:
It's for live preview. Casing works the same with SwiftUI but for some reason it's convention to name your preview structs like that :/